src/pages/rss.xml.ts

```diff
@@ -5,11 +5,28 @@ import { SITE_TITLE, SITE_DESCRIPTION } from "../consts";
 
 export const GET: APIRoute = async ({ site }) => {
   const posts = await getCollection("blog");
+
+  const sorted_posts = posts.sort((a, b) => Number(b.data.pubDate) - Number(a.data.pubDate))
+
+  const site_url =
+  process.env.NODE_ENV === 'production'
```

Comment:
prefer `PUBLIC_VERCEL_ENV`, just for consistency with `BaseHead`

Reply:
Resolved in [d6ba539](https://github.com/purduehackers/blog/pull/26/commits/d6ba53988061b6254564b1d9930c1f3dffa13931)